d2/src/main/java/com/linkedin/d2/balancer/simple/SimpleLoadBalancerState.java

```diff
@@ -679,6 +679,7 @@ public void setDelayedExecution(long delayedExecution)
                                                   Map<URI, TrackerClient> potentialClients)
   {
     SubsettingStrategy<URI> subsettingStrategy = _subsettingStrategyFactory.get(serviceName, minClusterSubsetSize, partitionId);
+    final int LOG_SUBSET_MAX_SIZE = 20;
```

Comment:
This can be static.

BTW how do we decide the size of 20, if the default subset size is 10, I would imagine something around 10 would cover many use cases.

Reply:
Changed to static. 

Since this is a debug level log, leaving a little room may be useful for experimenting and tuning this config. We don't expect this to be in the actual production log.